README.md

```diff
@@ -23,10 +23,10 @@ module.exports = {
             user: 'user',
             password: 'password',
             charset: 'utf8',
-            name: 'ghost'
+            database: 'ghost'
         }
     },
-    migrationPath: process.cwd() + '/core/server/data/migrations',
+    migrationPath: process.cwd() + '/current/core/server/data/migrations',
```

Comment:
This is just an example and it depends on the context.
If you are using the Ghost CLI, it makes sense to use `current`, but if not it depends on the folder structure. 
Maybe leave the example as is and add a Note below the example that if using the Ghost-CLI you have to use this path.

Reply:
I see. By the way I had it without the `current` and ran `knex-migrator init` which just said something like 'Done' without actually doing anything.